core-ui/src/shared/components/K8sResourceSelect.js

```diff
@@ -98,6 +98,7 @@ export function K8sResourceSelect({
         })}
```

Comment:
I cannot add a comment there but shouldn't we switch the selectionType to 'manual' (line 93)? this one doesn't work good